deploy/helm/kubecf/assets/scripts/jobs/cf-cli-6-linux/setup_internal_security_group.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+set -o errexit -o nounset -o xtrace
+
+export PATH="${CF_CLI_PATH}:${PATH}"
+
+until curl --insecure --fail --silent --head "${CF_API}"; do
+    sleep 1
+done
+
+cf api --skip-ssl-validation "${CF_API}"
```

Comment:
Can we do anything to avoid the `--skip-ssl-validation`?

Reply:
+1, we can't leave insecure in 
also `--silent` is a bad idea

Reply:
@viovanov I guess the `--silent` is regarding the `curl` command. That command will be triggered multiple times, and we don't want it to spam logs.

For the `--skip-ssl-validation`, I have a plan for it that ties with the commend above about using the internal hostname.

Reply:
Created https://github.com/cloudfoundry-incubator/kubecf/issues/451.